core/src/main/java/org/apache/calcite/sql/dialect/HiveSqlDialect.java

```diff
@@ -166,4 +173,36 @@ public HiveSqlDialect(Context context) {
     }
     return super.getCastSpec(type);
   }
+
+  /**
+   * Rewrite SINGLE_VALUE.
+   *
+   * <pre>In Hive, ASSERT_TRUE(false) will throw assertion failed exception.</pre>
```

Comment:
I would swap this <pre> with the next one.
It makes it look like you are solving a problem related to ASSERT_TRUE.

Reply:
Fixed.